docs/coding-guidelines/updating-ref-source.md

```diff
@@ -14,5 +14,5 @@ This document provides the steps you need to take to update the reference assemb
 These steps can also be applied to some unique assemblies which depend on changes in System.Private.Corelib coming from [coreclr](https://github.com/dotnet/coreclr) (partial facades like [System.Memory](https://github.com/dotnet/corefx/blob/83711167ee74d2e87cf2d5ed3508c94044bb7edc/src/System.Memory/src/System.Memory.csproj#L6), for example).
 1) Build coreclr release.
 2) Build corefx release with coreclr bits (see (// TODO //)) for more details).
-3) Run `msbuild /t:GenerateReferenceSource /p:ConfigurationGroup=Release` from the System.Runtime/ref directory.
+3) Run `msbuild /t:GenerateReferenceSource /p:Configuration=Release` from the System.Runtime/ref directory.
```

Comment:
Please update this document by removing 1) and 2) and update 3) to use `dotnet build` instead of `msbuild`.